code/_onclick/hud/fullscreen.dm

```diff
@@ -72,6 +72,8 @@
 		if(screen.needs_offsetting)
 			screen.plane = GET_NEW_PLANE(initial(screen.plane), offset)
 
+INITIALIZE_IMMEDIATE(/atom/movable/screen/fullscreen)
```

Comment:
Yes, otherwise there are GC failures caused by the outfit_sanity unit test. The only reason that error is not happening before this PR are because none of the outfits it tests have the specific combination of tint level and flags_inv to trigger it. This change just happens to make that bug always happen instead of relying on a particular combination of clothing characteristics.